packages/legend-studio/src/components/editor/edit-panel/connection-editor/RelationalDatabaseConnectionEditor.tsx

```diff
@@ -604,6 +636,19 @@ const SnowflakePublicAuthenticationStrategyEditor = observer(
   },
 );
 
+const GCPApplicationDefaultCredentialsAuthenticationStrategyEditor = observer(
```

Comment:
If you just return nothing like this, perhaps, you don't need this at all, the top handler will render nothing when the auth strategy is not supported